pkg/kubectl/cmd/explain.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2014 The Kubernetes Authors.
+Copyright 2016 The Kubernetes Authors.
```

Comment:
This date stays the same.  It's the date the file was created, not the date of the last modification.